src/TracingDriverManager.php

```diff
@@ -65,4 +71,29 @@ public function createNullDriver()
     {
         return new NullTracer();
     }
+
+    /**
+     * @return BinarySampler|PercentageSampler
+     * @throws InvalidArgumentException
+     */
+    protected function getZipkinSampler()
+    {
+        $samplerClassName = $this->config->get('tracing.zipkin.sampler_class');
```

Comment:
x2 issues with the new `sampler_class` setting:
- It's not mentioned in the `v2.3.2` release that this setting has been added
- The setting doesn't have a default value, leading to errors when not populated